security_groups.tf

```diff
@@ -18,21 +19,47 @@ resource "aws_security_group" "eq_alb" {
   }
 
   tags {
-    Name        = "${var.env}-eq-alb-access"
+    Name        = "${var.env}-eq-alb"
     Environment = "${var.env}"
   }
 }
 
-resource "aws_security_group" "eq_ecs" {
-  name        = "${var.env}-eq-ecs"
-  description = "Allow access from the internet or WAF"
+resource "aws_security_group" "eq_alb_ons_access" {
+  name        = "${var.env}-eq-alb-access-from-ons"
+  description = "Allow access to ALB from the ONS"
+  vpc_id      = "${var.vpc_id}"
+  count       = "${length(var.ons_access_ips) > 0 ? 1 : 0}"
+
+  ingress {
+    from_port   = "443"
+    to_port     = "443"
+    protocol    = "tcp"
+    cidr_blocks = "${var.ons_access_ips}"
+  }
+
+  egress {
+    from_port   = 0
+    to_port     = 0
+    protocol    = "-1"
+    cidr_blocks = ["0.0.0.0/0"]
+  }
+
+  tags {
+    Name        = "${var.env}-eq-alb"
```

Comment:
I think we should set the `Name` tag to `{env}-eq-ecs`, as:
- The security group name captures the specific name
- We'll be able to easily identify all the security groups for the ECS cluster